File: maskinporten-client/src/main/java/no/ks/fiks/maskinporten/error/MaskinportenClientTokenRequestException.java
package no.ks.fiks.maskinporten.error;

/**
 * Used for client errors i.e when the response status for a token request is in the 4xx range
 */
public class MaskinportenClientTokenRequestException extends MaskinportenTokenRequestException {

    public MaskinportenClientTokenRequestException(String message, int statusCode, String maskinportenError) {
        super(message, statusCode, maskinportenError);
    }
}

File: maskinporten-client/src/main/kotlin/no/ks/fiks/maskinporten/error/MaskinportenClientTokenRequestException.kt
package no.ks.fiks.maskinporten.error

/**
 * Used for client errors i.e when the response status for a token request is in the 4xx range
 */
class MaskinportenClientTokenRequestException(message: String?, statusCode: Int, maskinportenError: String) :
    MaskinportenTokenRequestException(message, statusCode, maskinportenError)